src/scss/layout/_header.scss

```diff
@@ -1,6 +1,107 @@
 body{
-    background-color: rgb(85, 175, 202);
+  background-color: rgb(85, 175, 202);
 }
-h1{
-    color: green;
+
+.header{
+  height: 143.7vw;
+
+  @include phone{
+    background-image: url('/build/img/mobile/image-header.jpg');
+    background-position: top;
+    background-repeat: no-repeat;
```

Comment:
you can do these lines in only one: `background-image: url(image) top no-repeat
the only attribute that is okay to be separated is background-size, because if you add that to one line safari does not recognize it